storage/src/mmr/mem.rs

```diff
@@ -295,7 +295,6 @@ impl<H: CHasher> Mmr<H> {
             if peak_pos < pos {
                 continue;
             }
-
```

Comment:
nit: for consistency, prefer we retain this `\n`

Reply:
done